.travis.yml

```diff
@@ -1,18 +1,21 @@
-sudo: false
+sudo: required
+dist: trusty
 notifications:
   email: false
 language: python
-addons:
-  apt:
-    packages:
-      - libgdal1h
-      - libgdal1-dev
-      - gdal-bin
 env:
-  - TOX_ENV=py27
-  - TOX_ENV=py33
-  - TOX_ENV=py34
-  - TOX_ENV=coveralls
+  global:
+    - CPLUS_INCLUDE_PATH=/usr/include/gdal
+    - C_INCLUDE_PATH=/usr/include/gdal
```

Comment:
I'll just point out these are both set to the same value. I don't necessarily feel a change is needed, but thought I'd mention it in case it was not intended.